content/en/docs/concepts/security/index.md

```diff
@@ -617,346 +615,290 @@ services in an Istio Mesh. It features:
     >}}
 
 The above diagram shows the basic Istio authorization architecture. Operators
-specify Istio authorization policies using `.yaml` files. Once deployed, Istio
-saves the policies in the `Istio Config Store`.
+specify Istio authorization policies using `.yaml` files.
 
-Pilot watches for changes to Istio authorization policies. It fetches the
-updated authorization policies if it sees any changes. Pilot distributes Istio
-authorization policies to the Envoy proxies that are colocated with the
-service instances.
+Galley watches for changes to Istio authorization policies and fetches the
```

Comment:
I would remove this entire paragraph. In general, we want to avoid architectural discussions in the docs and focus on features.

Reply:
Removed.